### PR TITLE
Joseph/awsstoragelens

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

Dashboard for storagelens. This data comes in via cloudwatch metrics, so the integration is already in place.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/amazon-s3-storagelens/amazon-s3-storagelens.json [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+              "row": 1,
+              "width": 2,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![New Relic logo](https://lh6.googleusercontent.com/eY3HD-U5lVs38NUsZIgraXMVSuMDfopVCUihlytZ9e6-tF_na-3ENIBte7l05mfPrpp7MXUY_vnmyeS5QlDN1xqm00HrHEoxPkHIAi1BERTdjZZKAraFeIzJ1I6ts_hYnLzXes1a)\n# AWS S3 Storage Lens Dashboard"
```
Comment (pkudikyala) [inline dashboards/amazon-s3-storagelens/amazon-s3-storagelens.json:22]:
```suggestion
              "text": "![AWS S3 Storage Lens logo](https://lh6.googleusercontent.com/eY3HD-U5lVs38NUsZIgraXMVSuMDfopVCUihlytZ9e6-tF_na-3ENIBte7l05mfPrpp7MXUY_vnmyeS5QlDN1xqm00HrHEoxPkHIAi1BERTdjZZKAraFeIzJ1I6ts_hYnLzXes1a)\n# AWS S3 Storage Lens Dashboard"
```

**Review by pkudikyala — CHANGES_REQUESTED** [review]
Could you remove the SINCE condition in each query, if it is not required.

**Comment by jcountsNR** [comment]
> Could you remove the SINCE condition in each query, if it is not required.

The 'SINCE' in this case is actually crucial. These metrics come in only once per day, so the query needs to be exactly 1 day unless they are timeseries, then they need to be a few days grouped by 1 day.

**Comment by pkudikyala** [comment]
> > Could you remove the SINCE condition in each query, if it is not required.
> 
> The 'SINCE' in this case is actually crucial. These metrics come in only once per day, so the query needs to be exactly 1 day unless they are timeseries, then they need to be a few days grouped by 1 day.

Okay, thanks for the clarification.

**Review by pkudikyala — APPROVED** [review]
LGTM!

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM

**Pull request merged**